tx-pool/src/pool.rs

```diff
@@ -380,11 +388,14 @@ impl TxPool {
         &self,
         proposal_id: &ProposalShortId,
     ) -> Option<TransactionView> {
-        self.get_tx_from_pool(proposal_id).cloned().or_else(|| {
-            self.committed_txs_hash_cache
-                .peek(proposal_id)
-                .and_then(|tx_hash| self.snapshot().get_transaction(tx_hash).map(|(tx, _)| tx))
-        })
+        self.get_tx_from_pool(proposal_id)
+            .cloned()
+            .or_else(|| self.conflicts_cache.peek(proposal_id).cloned())
```

Comment:
When using `get_tx_with_cycles`, should it use `peek` or `get`?

> Unlike `get`, `peek` does not update the least recently used (LRU) list, so the position of the key will remain unchanged.

Reply:
I think we should use `peek` here (the difference is very trivial),
`get_tx_from_pool_or_store` is with `&self`, if use `get` we need `mut self`, 
I guess the same reason applied to `committed_txs_hash_cache.peek(..)` .